src/eradiate/scenes/biosphere/__init__.pyi

```diff
@@ -1,4 +1,12 @@
 from ._canopies import wellington_citrus_orchard as wellington_citrus_orchard
+from ._canopy_loader import load_scenario as load_scenario
+from ._rami_scenarios import (
+    ActualCanopies as ActualCanopies,
```

Comment:
For context, maybe it would be better to prefix these enum names, *e.g.* `RAMIActualCanopies`?